neo/rawio/cedrawio.py

```diff
@@ -0,0 +1,177 @@
+"""
+Class for reading data from CED (Cambridge Electronic Design)
+http://ced.co.uk/
+
+This read *.smrx (and *.smr) from spike2 and signal software.
+
+Note Spike2RawIO/Spike2IO is the old implementation in neo.
```

Comment:
Would it be worth also adding a reference in the Spike2IO to this one as an alternative implementation?

Reply:
@samuelgarcia And what is your opinion regarding this proposal?